<launch>
    <node pkg="hrl_clickable_display" type="arrow_overlay" name="arrow_overlay" respawn="true" output="screen">
        <remap from="/image_in" to="/kinect_head/rgb/image_color" />
        <remap from="/image_out" to="/arrow_overlaid" />
        <param name="shaft_size" type="double" value="0.1" />
        <param name="head_width" type="double" value="0.03" />
        <param name="head_length" type="double" value="0.03" />
        <param name="debug_mode" type="bool" value="true" />
    </node>

    <node pkg="pixel_2_3d" type="pixel_2_3d" name="pixel_2_3d" output="screen" respawn="true">
        <remap from="image" to="/kinect_head/rgb/image_color"/>
        <remap from="point_cloud" to="/kinect_head/rgb/points"/>
        <remap from="l_mouse_click" to="/clickable_display/l_mouse_click"/>
        <param name="normal_radius" type="double" value="0.025" />
    </node>
        
</launch>
